cmd/sls/sls_hardware.go

```diff
@@ -479,7 +479,7 @@ func doHardwareSearch(w http.ResponseWriter, r *http.Request) {
 				pdet := base.NewProblemDetails("about: blank",
 					"Internal Server Error",
```

Comment:
This probably shouldnt say 'internal server error' still.

Reply:
I will fix that.